bin.src/exportGraphInputs.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env python
```

Comment:
Some sort of comment at the top of this file to explain what it's for would be useful.

And the regular copyright boilerplate, I guess.

Reply:
Indeed, none of the files here have the requisite boilerplate, and the repository doesn't contain the required `LICENSE` and `COPYRIGHT` files.

Reply:
Okay, @ctslater didn't put them in his initial attempt and I forgot.